src/vss_tools/vspec/vssexporters/vss2samm/helpers/namespaces.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2024 Contributors to COVESA
+#
+# This program and the accompanying materials are made available under the
+# terms of the Mozilla Public License 2.0 which is available at
+# https://www.mozilla.org/en-US/MPL/2.0/
+#
+# SPDX-License-Identifier: MPL-2.0
+
+from rdflib import URIRef
+from vss_tools import log
+from .. config import config as cnfg
+
+
+def get_vspec_uri(node_name: str):
+    return URIRef(f'{samm_output_namespace}{node_name}')
+
+
+def get_node_name_from_vspec_uri(node_uri: URIRef):
+    return node_uri.replace(samm_output_namespace, '')
+
+
+def get_unit_uri(unit_name: str):
+    return URIRef(f"{samm_base_namespace}:unit:{cnfg.SAMM_VERSION}#{unit_name}")
+
+
+log.debug(
```

Comment:
Usually running things on import time of files is considered to be an antipattern  except when initializing globals I guess